packages/lib/src/components/results/ChartComponent.wc.svelte

```diff
@@ -391,15 +394,23 @@
      * @param responseStore - the response store
      */
     const setChartData = (responseStore: ResponseStore): void => {
-        if (responseStore.size === 0) return;
+        if (responseStore.size === 0) {
```

Comment:
The logic is not correct, when the responseStore is empty it should be the "empty charts" look like the initial load. If the Store has results but not regarding this Diagramm then it should show the text